cli/ops.rs

```diff
@@ -74,18 +76,44 @@ fn empty_buf() -> Buf {
   Box::new([])
 }
 
+fn slice_u8_as_i32(bytes: &[u8]) -> &[i32] {
+  let p = bytes.as_ptr();
+  // Assert pointer is 32 bit aligned before casting.
+  assert_eq!((p as usize) % std::mem::align_of::<i32>(), 0);
+  #[allow(clippy::cast_ptr_alignment)]
+  let p32 = p as *const i32;
+  unsafe { std::slice::from_raw_parts(p32, bytes.len() / 4) }
```

Comment:
The purpose of this is to not have to make two conversions from `&[u8]` into `&[i32]` (for `has_minimal_token` and `Record::from` ... probably those conversions compile into nothing tho)

Reply:
After thinking about it, i agree, I removed the "From" trait on record (because it can fail) and it's now called "parse_min_record" and I added a test for it.